zubhub_frontend/zubhub/package.json

```diff
@@ -4,10 +4,13 @@
   "private": true,
   "dependencies": {
     "@date-io/date-fns": "^1.3.13",
+    "@emotion/react": "^11.9.0",
+    "@emotion/styled": "^11.8.1",
     "@material-ui/core": "^4.11.2",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.61",
     "@material-ui/pickers": "^3.2.10",
+    "@mui/material": "^5.6.2",
```

Comment:
we are using `material-ui` and not `mui`. There is tooltip being used already in the project, maybe we use that instead of introducing another new package? This is usually good to keep consistent UI and reduce intermixing of styles. Here is a example of tooltip used currently : https://github.com/unstructuredstudio/zubhub/blob/master/zubhub_frontend/zubhub/src/views/edit_profile/EditProfile.jsx#L25